cdm-test/src/test/java/ucar/nc2/jni/netcdf/TestNc4IospWriting.java

```diff
@@ -146,5 +165,56 @@ private boolean compare(NetcdfFile nc1, NetcdfFile nc2, boolean showCompare, boo
   }
 
 
+  /////////////////////////////////////////////////
 
+  // Demonstrates GitHub issue #191. Unignore when we have a fix in place.
+  @Test
+  @Ignore
+  public void writeEnumType() throws IOException {
+    // NetcdfFile's 0-arg constructor is protected, so must use NetcdfFileSubclass
+    NetcdfFile ncFile = new NetcdfFileSubclass();
+
+    // Create shared, unlimited Dimension
+    Dimension timeDim = new Dimension("time", 3, true, true, false);
+    ncFile.addDimension(null, timeDim);
+
+    // Create a map from integers to strings.
+    Map<Integer, String> enumMap = new HashMap<>();
+    enumMap.put(18, "pie");
+    enumMap.put(268, "donut");
+    enumMap.put(3284, "cake");
+
+    // Create EnumTypedef and add it to root group.
+    EnumTypedef dessertType = new EnumTypedef("dessertType", enumMap, DataType.ENUM4);
+    ncFile.getRootGroup().addEnumeration(dessertType);
+
+    // Create Variable of type dessertType.
+    Variable dessert = new Variable(ncFile, null, null, "dessert", DataType.ENUM2, "time");

```

Comment:
Fixed: ff02f198e74b87a86ecdaff0645ca6ab5ed8473a

It should be ENUM2, actually; the storage is shorts.